pyproject.toml

```diff
@@ -0,0 +1,130 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "dbt-databricks"
+dynamic = ["version"]
+description = "The Databricks adapter plugin for dbt"
+readme = "README.md"
+license = "Apache-2.0"
+requires-python = ">=3.9"
+authors = [{ name = "Databricks", email = "feedback@databricks.com" }]
+classifiers = [
+    "Development Status :: 5 - Production/Stable",
+    "License :: OSI Approved :: Apache Software License",
+    "Operating System :: MacOS :: MacOS X",
+    "Operating System :: Microsoft :: Windows",
+    "Operating System :: POSIX :: Linux",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
+    "Programming Language :: Python :: 3.11",
+    "Programming Language :: Python :: 3.12",
+]
+dependencies = [
```

Comment:
These dependencies are the ones required to build the package.  The default env dependencies specified below are what we want installed when we're working in IDE/using the CLI.

Reply:
are the dependencies exactly same as previous version? If not, shall we bump up the version